.travis.yml

```diff
@@ -114,6 +118,9 @@ matrix:
       #- env: CCOMPILER='clang-3.8' CXXCOMPILER='clang++-3.8' BUILD_TYPE='Release' BUILD_SHARED_LIBS=ON
 
 before_install:
+  - if [[ ! -z $APT_ARCH ]] ; then sudo dpkg --add-architecture $APT_ARCH ; fi
```

Comment:
This is interesting. We should talk to travis about adding this flag to their builds.